chromium_src/chrome/test/base/chrome_unit_test_suite.cc

```diff
@@ -3,9 +3,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#include "chrome/test/base/chrome_unit_test_suite.h"
+
 #include "brave/common/brave_content_client.h"
+#include "brave/common/resource_bundle_helper.h"
 #include "brave/test/base/testing_brave_browser_process.h"
+#include "chrome/browser/profiles/profile_shortcut_manager.h"
+
+#define DisableForUnitTests \
+  DisableForUnitTests();    \
+  brave::InitializeResourceBundle
```

Comment:
this doesn't seem like the right way to handle this, `brave::InitializeResourceBundle` should always be called along with upstream

Reply:
I see what's going on now, but I think a better override would be for `InitSharedInstanceWithLocale`

Reply:
Changed in 9bb3f332a666d679870d7bcb6fb8421672e521e5